source/libraries/ContractDeployer.ts

```diff
@@ -42,11 +42,12 @@ export class ContractDeployer {
     }
 
     public async deploy(): Promise<boolean> {
-        this.testAccounts = await generateTestAccounts(this.testAccountSecretKeys);
+        const accounts = await this.ethjsQuery.accounts();
+        this.testAccounts = accounts;
```

Comment:
Do you think it's worth keeping `generateTestAccounts` for unit tests, or should I remove it?

Reply:
Also, should I remove the `secretKeys` being passed into the constructor along with `this.testAccountSecretKeys`?

Reply:
We may end up bringing back `generateTestAccounts` for unit tests, but we can always just pull them out of git history.  Same with the secret keys I think.  So no need to keep them around for now I think.